### PR TITLE
Use own list of source templates for testing pursposes to prevent missing template

### LINKED ISSUE #552: [cmd-line]: CI Jenkins job failing on branch feature/cmd-line
**Describe the bug**
CI job fails due to falining shell unit tests as part of Jenkins job on branch feature/cmd-line.

**To Reproduce**
1. On local linux environment, execute following:
```
gh repo clone pharo-project/pharo-launcher
cd pharo-launcher
git checkout feature/cmd-line
PHARO=100 ARCHITECTURE=64 ./build.sh prepare
PHARO=100 ARCHITECTURE=64 ./build.sh test
```
2. Console output prints following error:
```
testLauncherProcessListCommandWhenImageIsLaunchedShouldReturnOneImage
NotFound: Please enter a correct local image.
ASSERT:Actual: "", expected: "PhLTestImage". Not found:<PhLTestImage>
shunit2:ERROR testLauncherProcessListCommandWhenImageIsLaunchedShouldReturnOneImage() returned non-zero return code.
```
**Expected behavior**
Shell unit tests pass.

**Version information:**
 - OS: Linux, x64
 - Version: P10, 64bit

**Additional context**
Error occurs due to fact that template list is outdated and image template for P10 stable is not found. 
- Either fix by removing ~/Pharo/sources.list in one time setup of shell unit test, missing file will trigger sources.list download.
- Or implement new command to refesh image templates that would be part of one time setup (more work).

### PR DESCRIPTION
**Pull request by Bajger**
- fixes #552
- added sources list test setup and restore of original template list

### CODE REVIEW

**Comment by Bajger** [comment]
@demarey: Hi Christophe! Can you trigger CI for this PR? It seems that is is not running automatically.

File: test/PharoLauncherCommonFunctions.sh [hunk 1]
```diff
@@ -38,6 +38,18 @@ detectPharoLauncherImagePath() {
 		exit 1;
 	fi
 }
+setupImageTemplateList () {
+    #using own image template list file to have same templates that are used for testing
+    pushd .. > /dev/null
+    cp ~/Pharo/sources.list ~/Pharo/sources.list.original
```
Comment (Bajger) [inline test/PharoLauncherCommonFunctions.sh]: I'm not sure it is the best way to hardcode launcher target installation directory (where sources.list are located), but I don't know how to detect it properly.

**Comment by demarey** [comment]
Hi David,
Here is the CI log for this PR: https://ci.inria.fr/pharo-ci-jenkins2/job/PharoLauncher-Pipeline/view/change-requests/job/PR-555/2/console

<blockquote><div><strong><a href="https://ci.inria.fr/pharo-ci-jenkins2/job/PharoLauncher-Pipeline/view/change-requests/job/PR-555/2/console">PharoLauncher-Pipeline » PR-555 #2 Console [Jenkins]</a></strong></div></blockquote>

**Comment by Bajger** [comment]
Hi it is hard to determine, if some tests passed or not: 
```
Ran [1;36m4[0m tests.

[1;31mFAILED[0m ([1;31mfailures=1[0m)
```
How can be this fixed? Any idea?

**Comment by Bajger** [comment]
It seems: there is something wrong with the startup of image: 
```
testLauncherKillAllCommandWithOneImageLaunchedShouldKillAll
WARN: 

Assertion failure at SDL_CreateTexture_REAL (/home/travis/build/OpenSmalltalk/opensmalltalk-vm/build.linux64x64/third-party/SDL2-2.0.7/src/render/SDL_render.c:498), triggered 1 time:
  'renderer && renderer->magic == &renderer_magic'

Abort/Break/Retry/Ignore/AlwaysIgnore? [abriA] : [1;31mASSERT:[0mActual: "16138 /builds/Pharo/vms/100-x64/lib/pharo --headless /tmp/pharo-launcher-tests-launch-image-command.c16e04f7-a39a-0d00-a996-1a4809235d18/PharoLauncher.image --no-quit 16166 /builds/Pharo/vms/100-x64/lib/pharo --headless /tmp/pharo-launcher-tests-launch-image-from-disk-command.f39129f7-a39a-0d00-a99b-9f3009235d18/PharoLauncher.image --no-quit", expected: "PhLTestImage". Not found:<PhLTestImage>
```

**Comment by demarey** [comment]
> Hi it is hard to determine, if some tests passed or not:
> 
> ```
> Ran �[1;36m4�[0m tests.
> 
> �[1;31mFAILED�[0m (�[1;31mfailures=1�[0m)
> ```
> 
> How can be this fixed? Any idea?

@Bajger I fixed it in https://github.com/pharo-project/pharo-launcher/commit/a28d85bb0785ad3fcb4478cba71e7f4f194eea64

**Comment by demarey** [comment]
I also fixed a pb in shunit2 that run oneTimeTearDown() twice for the last test (see https://github.com/pharo-project/pharo-launcher/commit/3a7072211ad8c0c07d770c40c796035d5182f1ec)

**Comment by Bajger** [comment]
@demarey PR looks ok now, CI tests green, output of Shunit tests looks fine. You can squash & merge now!

**Pull request merged**